shared/src/e2e/config.ts

```diff
@@ -66,13 +68,44 @@ const configFields: ConfigFields = {
         description:
             'If true, include admin onboarding tests, which assume none of the admin onboarding steps have yet completed on the instance. If those steps have already been completed, this test will fail.',
     },
+    testUserPassword: {
+        envVar: 'TEST_USER_PASSWORD',
+        description:
+            'The password to use for any test users that are created. This password should be secure and unguessable when running against production Sourcegraph instances.',
+    },
+    noCleanup: {
+        envVar: 'NO_CLEANUP',
+        parser: parseBool,
+        description:
+            "If true, regression tests will not clean up users, external services, or other resources they create. Set this to true if running against a dev instance (as it'll mamke test runs faster). Set to false if running against production",
```

Comment:
typo:

```suggestion
            "If true, regression tests will not clean up users, external services, or other resources they create. Set this to true if running against a dev instance (as it'll make test runs faster). Set to false if running against production",
```

maybe better:

```suggestion
            "If true, regression tests will not clean up users, external services, or other resources they create. Set this to true if running against a dev instance (as it'll make tests run faster). Set to false if running against production",
```